controller/src/beerocks/master/beerocks_master_main.cpp

```diff
@@ -218,6 +218,33 @@ static void fill_master_config(son::db::sDbMasterConfig &master_conf,
     } else {
         master_conf.load_steer_on_vaps = std::string(load_steer_on_vaps);
     }
+
+    if (!beerocks::bpl::cfg_get_persistent_db_enable(master_conf.persistent_db)) {
+        LOG(DEBUG) << "failed to read persistent db enable, setting to default value: "
+                   << bool(DEFAULT_PERSISTENT_DB);
+        master_conf.persistent_db = (DEFAULT_PERSISTENT_DB == 1);
+    }
+    if (!beerocks::bpl::cfg_get_clients_persistent_db_max_size(
+            master_conf.clients_persistent_db_max_size)) {
+        LOG(DEBUG)
+            << "failed to read max number of clients in persistent db, setting to default value: "
+            << int(DEFAULT_CLIENTS_PERSISTENT_DB_MAX_SIZE);
+        master_conf.clients_persistent_db_max_size = DEFAULT_CLIENTS_PERSISTENT_DB_MAX_SIZE;
```

Comment:
Well, you already let the user decide what to do in case of failure. So this enforce what I said on one of the previous comments. :)

Reply:
you need to differentiate between `failure` and `not-configured` - even if currently the user uses the same default as the implementation for now